Sources/App/Controllers/PackageController+Queries.swift

```diff
@@ -74,9 +74,10 @@ extension PackageController {
 
             static func query(on database: Database, owner: String, repository: String) -> EventLoopFuture<(PackageInfo)> {
                 Joined3<Package, Repository, Version>
-                    .query(on: database, version: .defaultBranch)
-                    .filter(Repository.self, \.$owner, .custom("ilike"), owner)
-                    .filter(Repository.self, \.$name, .custom("ilike"), repository)
+                    .query(on: database, owner: owner, repository: repository, version: .defaultBranch)
+                    .field(Repository.self, \.$owner)
+                    .field(Repository.self, \.$name)
+                    .field(Version.self, \.$packageName)
```

Comment:
I notices while doing the maintainerInfo update that the "builds route" query was also pulling in unnecessary amounts of data.